Hackaton/readme.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
@LiliamLeme Why a blank file? and why txt and not md?

Reply:
You can ignore this file. I will exclude from the PR. I just created as a test when I create the folder